tests/system/src/test/java/test/robot/javafx/stage/WrongStageFocusWithApplicationModalityTest.java

```diff
@@ -85,6 +89,16 @@ private static void keyPress(KeyCode code) throws Exception {
         });
     }
 
+    private static void mouseClick() {
+        Util.runAndWait(() -> {
+            robot.mouseMove((int) (alert.getX()+alert.getWidth()/2),
+                    (int) (alert.getY()+alert.getHeight()/2));
```

Comment:
Minor: we usually put spaces around binary operators.

Reply:
Fixed